README.md

```diff
@@ -115,9 +113,11 @@ Patch releases are defined as bug, performance, and security fixes. They do not
 
 Minors are the addition and refinements of APIs or subsystems. They do not change APIs or introduce backwards compatible breaking changes. These are mostly completely additive releases.
 
+The only exception to the versioning rules for Minors is when dealing with fixes for security vulnerabilities on Release branches other than master. For instance, if the current master is on major version 3.x, and a security vulnerability is discovered within the the 2.x release stream, then any fix for the security vulnerability, regardless of whether the fix introduces a backwards compatible breaking change, must result in a Minor version increase within the 2.x release stream (i.e. if the highest version in the 2.x stream was 2.6.7, then the security fix would cause the stream to bump to 2.7.0.) and the change in behavior must be clearly documented within the Pull Request.
```

Comment:
Not necessarily. It works *best* if LTS is cut on the last minor (in general I think there should only be one LTS per Major.Minor unless significant issues come up that force another one). But this would allow multiple LTS candidates per minor stream if absolutely necessary.